editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -199,6 +199,21 @@ void SpriteFramesEditor::_paste_pressed() {
 	undo_redo->commit_action();
 }
 
+void SpriteFramesEditor::_copy_pressed(){
+
+	ERR_FAIL_COND(!frames->has_animation(edited_anim));
+
+	if(tree->get_current() < 0)
```

Comment:
Not sure why `clang-format` did not complain, but this doesn't follow our style guide, it should be:
```
if (tree->get_current() < 0)
	return;
Ref<Texture> r = frames->get_frame(edited_anim, tree->get_current());
if (!r.is_valid()) {
	return;
}
```

Can you amend the commit with `git commit --amend` and update the PR with `git push --force`? See http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase

Reply:
See http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html regarding the code style and how to use `clang-format` locally.

Reply:
clang-format config error...
![capture](https://user-images.githubusercontent.com/24988459/32052953-4d43565c-ba8c-11e7-9d4e-9a133538468e.PNG)

Reply:
What's your `clang-format` version? We use 3.9.1 currently.